mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -329,7 +329,8 @@ class MultilayerPerceptronClassificationModel private[ml] (
 
   @Since("1.5.0")
   override def copy(extra: ParamMap): MultilayerPerceptronClassificationModel = {
-    copyValues(new MultilayerPerceptronClassificationModel(uid, layers, weights), extra)
+    val copied = new MultilayerPerceptronClassificationModel(uid, layers, weights)
```

Comment:
It doesn't seem that we have a "standard" way of doing this, i.e. some places do
```scala 
val copied = new X().setParent(parent)
``` 
and some places do 
```scala
val copied = ...
copyValues(...).setParent(parent)
```

But at least for this PR let's be consistent, so maybe change this one to be the first version like you have in the rest of the PR?